folium/plugins/semicircle.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+
+import json
+
+from branca.element import Figure, JavascriptLink
+
+from folium.map import Marker
+from folium.utilities import _validate_location
+
+from jinja2 import Template
+
+
+class SemiCircle(Marker):
+    """
+    Creates a Semicircle plugin to append into a map with
+    Map.add_plugin.
+    Use (direction and arc) or (startAngle and stopAngle)
+    Parameters
+    ----------
+    location: tuple of length 2, default None
+        The latitude and longitude of the marker.
+        If None, then the middle of the map is used.
+    radius: int, default 0
```

Comment:
Can you provide more sensible default arguments than zero? If the argument is optional use `None` instead of 0. But I think these are not optional right.